crates/solidity/testing/snapshots/cst_output/ContractMembers/separated_recovery/generated/0.4.11-failure.yml

```diff
@@ -64,15 +64,15 @@ Errors: # 10 total
        │                                                               ╰────────────── Error occurred here.
     ───╯
   - >
-    Error: Expected AddressKeyword or Identifier.
+    Error: Expected Identifier.
```

Comment:
See https://github.com/NomicFoundation/slang/pull/687#discussion_r1413795345 on why this regresses